pkg/loadbalancer/server.go

```diff
@@ -161,11 +161,13 @@ func (s *Server) EnsureLoadBalancerDeleted(ctx context.Context, clusterName stri
 
 // loadbalancer name is a unique name for the loadbalancer container
 func loadBalancerName(clusterName string, service *v1.Service) string {
-	hash := sha256.Sum256([]byte(loadBalancerSimpleName(clusterName, service)))
-	encoded := base32.StdEncoding.EncodeToString(hash[:])
-	name := constants.ContainerPrefix + "-" + encoded[:40]
-
-	return name
+	h := sha256.New()
+	_, err := io.WriteString(h, loadBalancerSimpleName(clusterName, service))
+	if err != nil {
+		panic(err)
```

Comment:
it seems this never return an error https://cs.opensource.google/go/go/+/refs/tags/go1.23.5:src/crypto/sha256/sha256.go;l=174-195